boards/adafruit_feather.json

```diff
@@ -0,0 +1,52 @@
+{
+  "build": {
+    "core": "earlephilhower",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_ADAFRUIT_FEATHER_RP2040 -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x2E8A",
+        "0x00C0"
+      ]
+    ],
+    "mcu": "rp2040",
+    "arduino": {
+      "earlephilhower": {
+        "variant": "adafruit_feather",
```

Comment:
Since the `earlephilhower` is the only available core, does it make sense to move it to the `build section`?

Reply:
Sorry, I wasn't clear enough, I meant to move only the `variant` field, others should be fine in their separate section as it's now.

Reply:
variant field is moved outside for earlephilhower-only boards in latest change.